Task2calculatorIF/src/Task2calculatorIF.java

```diff
@@ -0,0 +1,39 @@
+
+    import java.util.Scanner;
+
+    public class Task2calculatorIF {
+        public static void main(String[] args) {
+            String ProgramName = "Calculator";
+            System.out.println(ProgramName);
+            Scanner scan = new Scanner(System.in);
+            System.out.println("Enter the 1st number");
+            if (scan.hasNextInt()) {
```

Comment:
It works strange)
![image](https://user-images.githubusercontent.com/109154988/180759537-2e9f14f7-a0ec-4711-9ab7-a760cda3d512.png)

Reply:
As we should work with float/double numbers